### DISCUSSION TITLE
Is there any way to delete old binaries at once?

### DISCUSSION THREAD

**Discussion by ryoppippi** [aquaproj/aqua#2248] [Deprecated (Question and Answer)] [answered]
### Question

Is there any way to delete old binaries at once?
If there is such a command, I'd like to know

### Reference

*

**Comment by suzuki-shunsuke** [comment]
Thank you for your question.

> Is there any way to delete old binaries at once?

There is no way for it now.

- https://aquaproj.github.io/docs/guides/uninstall-packages/
- https://github.com/aquaproj/aqua/issues/538

But removing files under `AQUA_ROOT_DIR` directly by `rm` command isn't ideal, I'd like to reconsider it when I have time.

If you have any comments about this feature, please post them to https://github.com/aquaproj/aqua/issues/538

**Reply by ryoppippi** [reply]
I see. Thanks! I need that feature!!

**Reply by suzuki-shunsuke** [reply, marked as answer]
This feature was implemented at v2.11.0 🎉

- https://github.com/aquaproj/aqua/releases/tag/v2.11.0
- https://github.com/aquaproj/aqua/issues/538
- https://aquaproj.github.io/docs/guides/uninstall-packages/
